src/System.Private.ServiceModel/tools/SelfHostedWcfService/ChannelExtensibilityService.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Samples.MessageInterceptor;
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace WcfService
```

Comment:
This file is in the wrong place.  All server-side .cs files live under the IISHostedWcfService\App_code folder and are included by the self-host project.  Was there some reason you could not put this in IISHostedWcfService?